### ISSUE TITLE
Gaps in outline of rounded rectangle 

### ISSUE THREAD

**Issue by dev7355608** [pixijs/graphics-smooth#11]
If the radius of the rounded rectangle is very close to, equal or larger than the maximum possible radius (`min(width, height) / 2`), there appear gaps in the outline:

![image](https://user-images.githubusercontent.com/31905376/139583436-295878f0-eb8f-417a-81c0-811ed590f975.png)

```js
new PIXI.smooth.SmoothGraphics()
    .beginFill(0xff0000, 1.0)
    .lineStyle(3, 0x000000)
    .drawRoundedRect(100, 120, 100, 200, 50)
    .endFill()
    .beginFill(0x00ff00, 1.0)
    .lineStyle(3, 0x000000)
    .drawRoundedRect(250, 100, 100, 100, 50)
    .endFill()
    .beginFill(0xffff00, 1.0)
    .lineStyle(3, 0x000000)
    .drawRoundedRect(250, 250, 150, 100, 50)
    .endFill();
```

**Comment by ivanpopelyshev** [comment]
Most probably i didnt port https://github.com/pixijs/pixijs/pull/7516 . Let me try..

**Comment by ivanpopelyshev** [comment]
npm `0.1.17` , please check

**Comment by dev7355608** [comment]
Confirmed fixed in 0.0.17. Thanks!
